doc/src/manual/faq.md

```diff
@@ -22,7 +22,7 @@ On the other hand, language *interoperability* is extremely useful: we want to e
 
 ### How does Julia define its public API?
 
-The only interfaces that are stable with respect to [SemVer](https://semver.org/) of `julia`
+The only interfaces that are stable with respect to [SemVer](https://semver.org/) of Julia
```

Comment:
The language feels a bit weird here. Not your edits, but the sentence originally. Anything you can do to craft it better?

Reply:
Well... I propose to totally write this section in https://github.com/JuliaLang/julia/pull/50105, but the scope of that is way larger than this.

Reply:
How does this sound?
 > Base and standard library functionality described in [the documentation](https://docs.julialang.org/), which is not labeled as unstable (such as experimental and internal), is covered by [SemVer](https://semver.org/)." Functions, types, and constants are not part of the public API if they are not included in the documentation, _even if they have docstrings_.

Reply:
These are separate changes. I'm going to merge this as is, and do the reword in another PR